crates/collab/src/lib.rs

```diff
@@ -217,30 +217,30 @@ async fn build_blob_store_client(config: &Config) -> anyhow::Result<aws_sdk_s3::
     Ok(aws_sdk_s3::Client::new(&s3_config))
 }
 
-fn build_clickhouse_client(config: &Config) -> anyhow::Result<clickhouse::Client> {
```

Comment:
Could we keep this as returning a `Result` and just `ok()` it up above?

Will make it easier to debug if there's an environment variable missing.

Reply:
I went with this approach so that if you have clickhouse_url but not the other variables you get an immediate panic on startup; If you don't have clickhouse_url you'll get an error from the /events endpoint, but otherwise it'll silently be fine.

I think if we just ok'd it then it'd be harder to tell what's wrong (though maybe we should make the error message in /events clearer)